core/examples/json/record.json

```diff
@@ -32,8 +32,8 @@
     ]
   },
   "properties": {
-    "recordCreated": "2021-02-08",
-    "recordUpdated": "2021-02-08",
+    "created": "2021-02-08",
```

Comment:
All the values for created and updated in the examples are invalid, I think.
The spec says they are "datetime"s, but the examples are "date"s only. So I think it should be something like "2021-02-08T00:00:00Z"